### DISCUSSION TITLE
Checking of players on Android is disabled

### DISCUSSION THREAD

**Discussion by Bruzzler0** [pystardust/ani-cli#1179] [Q&A] [answered]
I have the problem that ani-cli does not work with termux. It comes the message 'Checking of players on Android is disabled' I have installed mpv installed. ![Screenshot_20230721_221021_Termux.jpg](https://github.com/pystardust/ani-cli/assets/106469103/f5549640-9843-4a0a-aebd-15e12c508738)

**Comment by port19x** [comment, marked as answer]
Hi, I assume you tried out ani-cli during the #1161 outage. 
Please check again, I don't think mpv is the issue here.
Also next time something doesn't work, prefer to open an issue, as we don't get notified about the creation of new discussions
